packages/bundle-source/src/script.js

```diff
@@ -79,8 +81,9 @@ export async function bundleScript(
     },
   );
 
-  const source = await makeBundle(powers, entry, {
-    dev,
+  let source = await makeFunctor(powers, entry, {
+    dev:
+      dev || moduleFormat === 'nestedEvaluate' || moduleFormat === 'getExport',
```

Comment:
Add a comment indicating that this automatic implication of `dev` is limited to "nestedEvaluate" and "getExport", and will not be extended to any future format.